package org.stepik.week4;

/**
 * @author Nika Zurbaevi
 */
public class QuestionAnswer_4_2_5 {
    /*
     * Вопрос: Рассмотрим пример кода:
     *
     *     try {
     *       // код, бросающий исключения
     *     } catch (RuntimeException e) {
     *       // ...
     *     } catch (Exception e) {
     *       // ...
     *     } catch (Error e) {
     *       // ...
     *     } finally {
     *       // ...
     *     }
     *
     * Предположим, что во время исполнения программы в теле блока try было
     * выброшено исключение типа java.lang.IllegalStateException. Какие блоки catch-finally будут выполнены?
     *
     *
     * Ответ:
     * 1) catch (RuntimeException e)
     * 2) finally
     *
     */
}
